pkg/services/ngalert/sender/router.go

```diff
@@ -116,13 +115,13 @@ func (d *AlertsRouter) SyncAndApplyConfigFromDatabase() error {
 
 		// We have no running sender and no Alertmanager(s) configured, no-op.
 		if !ok && len(cfg.Alertmanagers) == 0 {
-			d.logger.Debug("no external alertmanagers configured", "org", cfg.OrgID)
+			d.logger.Info("No external alertmanagers configured", "org", cfg.OrgID)
 			continue
 		}
 
 		// We have a running sender but no Alertmanager(s) configured, shut it down.
 		if ok && len(cfg.Alertmanagers) == 0 {
-			d.logger.Debug("no external alertmanager(s) configured, sender will be stopped", "org", cfg.OrgID)
```

Comment:
I think this can be info because this path will be hit when external alertmanager was configured but now it is not.